### PR TITLE
Modulesync: use voxpupuli helper gems

### PR DESCRIPTION
**Pull request by ekohl**

### CODE REVIEW

File: spec/spec_helper_acceptance.rb [hunk 1]
```diff
@@ -1,38 +1,20 @@
...
-
-require 'beaker-puppet'
-require 'beaker-rspec'
-require 'beaker/puppet_install_helper'
-require 'beaker/module_install_helper'
-
-run_puppet_install_helper unless ENV['BEAKER_provision'] == 'no'
-install_module_on(hosts)
-install_module_dependencies_on(hosts)
-install_module_from_forge('camptocamp-systemd', '>= 2.0.0 < 3.0.0')
```
Comment (ekohl) [inline spec/spec_helper_acceptance.rb:16]: https://github.com/theforeman/puppet-puppet/pull/710 will be needed to address this.

**Comment by mmoll** [comment]
please rebase :)

File: manifests/params.pp [hunk 2]
```diff
@@ -308,7 +308,7 @@
       # it reports its $osreleasemajor as 2, not 6.
       # thats why we're matching for '2' in both parts
       # Amazon Linux is like RHEL6 but reports its osreleasemajor as 2017 or 2018.
-      $osreleasemajor = regsubst($::operatingsystemrelease, '^(\d+)\..*$', '\1') # workaround for the possibly missing operatingsystemmajrelease
+      $osreleasemajor = regsubst($facts['os']['release']['full'], '^(\d+)\..*$', '\1') # workaround for the possibly missing operatingsystemmajrelease
```
Comment (ekohl) [inline manifests/params.pp:311]: I'm not sure about this, https://github.com/theforeman/puppet-puppet/pull/694 was intended to solve it in a cleaner way.

**Comment by mmoll** [comment]
merged, bedankt @ekohl!

**Pull request merged**